src/stylesheets/core/mixins/_usa-table-styles.scss

```diff
@@ -5,4 +5,32 @@
   .usa-table--borderless {
     @extend %usa-table--borderless;
   }
+  .usa-table--compact {
+    @extend %usa-table--compact;
+  }
+  .usa-table--striped {
+    @extend %usa-table--striped;
+  }
+  .usa-table--stacked {
+    @include at-media-max("mobile-lg") {
+      @include table-stacked-styles;
+    }
+  }
+  .usa-table--stacked-header {
+    @include at-media-max("mobile-lg") {
+      @include table-stacked-styles;
+      @include table-stacked-header-styles;
+    }
+  }
+  .width-mobile {
```

Comment:
Mostly, although I could see it as an acceptable way for someone to 'force' this display if they wanted to place it in a narrow column. Do you think it's inappropriate? We definitely do want a way to demo it on the docs page.

Reply:
That seems like it would be useful. Would that be in the guidance? Thanks for clarifying, btw.

You could add that breakpoint as a setting so users could force it when they need to.

Reply:
I really like allowing this as a utility. Smart addition.